Met Office doc updates for quality scale requirements (#46794)
home-assistant/home-assistant.github.io: source/_integrations/metoffice.markdown

{% include integrations/config_flow.md %}

{% configuration_basic %}
API Key:

>   description: The API key provided by the MetOffice when subscribing to the Site Specific Global Spot.

Reviewer: ```suggestion
  description: The API key provided by the Met Office when subscribing to the Site Specific Global Spot dataset.
```

Latitude:
  description: The latitude of the weather forecast location.
Longitude:
  description: The longitude of the weather forecast location.
{% endconfiguration_basic %}

Details about the <abbr title="Application Programming Interface">API</abbr> are available in the [DataHub API documentation](https://datahub.metoffice.gov.uk/docs/f/category/site-specific/overview). The [datapoint-python](https://github.com/EJEP/datapoint-python) library is used to retrieve data.

> ### Multiple Forecast Locations

Reviewer: ```suggestion
### Multiple forecast locations
```
We use [sentence-style capitalization](https://learn.microsoft.com/en-us/style-guide/capitalization), also in headings

If adding multiple forecasts, it is important to note that there are restrictions of the number of API queries allowed on the free tier. Currently that restricts this integration to a single location. Multiple locations require a paid plan.

The Latitude and Longitude of the forecast location are used to generate the unique ID for a given forecast. Therefore if adding multiple forecasts, each must have a unique location.

> ## Weather platform

Reviewer: Could you organize this into a Supported functionality section, with a subsection for Sensors?
https://github.com/home-assistant/home-assistant.io/blob/current/source/_integrations/_integration_docs_template.markdown?plain=1#L91

Author: Reorganized as requested.

- **Temperature** (°C): Current air temperature
- **Feels like temperature** (°C): Subjective temperature accounting for wind and humidity
- **Humidity** (%): Relative humidity
- **Probability of precipitation** (%): Chance of rain for the next hour
- **UV index**: UV index on standard international scale
- **Pressure** (Pa): Atmospheric pressure
- **Wind speed** (m/s): Current wind speed
- **Wind direction** (degrees): Wind direction in degrees
- **Wind gust** (m/s): Maximum wind gust speed
- **Visibility distance** (m): Distance of visibility
- **Weather**: One word summary of current weather conditions

Only probability of precipitation, temperature, weather and wind speed are enabled by default.

### Forecasts

The integration supports three types of forecasts:

- **Hourly forecast**: Available for the next 3 days with detailed conditions including temperature, precipitation, wind, and cloud coverage.
- **Twice daily forecast**: Available for the next 7 days, showing expected conditions for day and night.
- **Daily forecast**: Available for the next 7 days, showing daily high/low temperatures and midday conditions.

> ### Data updates

Reviewer: ```suggestion
## Data updates
```

> Weather data is automatically updated every 15 minutes from the MetOffice DataHub.

Reviewer: ```suggestion
Weather data is automatically updated every 15 minutes from the Met Office DataHub.
```

## Removing the integration

> This integration follows standard integration removal. No extra steps are required.

Reviewer: ```suggestion
This integration follows standard integration removal.
```
